packages/server/test/resources.e2e-spec.ts

```diff
@@ -1,3 +1,30 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+import { JwtAuthGuard } from './../src/auth/jwt-auth.guard';
+import { RolesGuard } from './../src/auth/roles.guard';
+
 describe('ResourcesController (e2e)', () => {
+    let app: INestApplication;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    })
+    .overrideGuard(JwtAuthGuard)
+    .useValue({ canActivate: () => true })
+    .overrideGuard(RolesGuard)
+    .useValue({ canActivate: () => true })
+    .compile();
```

Comment:
I would move it to a helper function, so login could be reused across tests. But, not many tests yet. Maybe later.